lib/config-array-factory.js

```diff
@@ -797,20 +806,35 @@ class ConfigArrayFactory {
      * @private
      */
     _loadExtendedBuiltInConfig(extendName, ctx) {
-        const { eslintAllPath, eslintRecommendedPath } = internalSlotsMap.get(this);
+        const {
+            eslintAllPath,
+            getEslintAllConfig,
+            eslintRecommendedPath,
+            getEslintRecommendedConfig
+        } = internalSlotsMap.get(this);
 
         if (extendName === "eslint:recommended") {
+            const name = `${ctx.name} » ${extendName}`;
+
+            if (getEslintRecommendedConfig) {
```

Comment:
Can we add in a check here to make sure getEslintRecommendedConfig is a function, and throw an error if not? I can picture this being difficult to debug if someone passes a nonfunction accidentally.

Reply:
Done.